### ISSUE TITLE
Support PRs from GitHub users outside the PolicyEngine organization

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/openfisca-us#768]
Currently some of the PR actions don't work when pushed from a fork, because they rely on secrets in the PolicyEngine organization to deploy to pip. Instead, our guidance is to push branches upstream directly to PolicyEngine. This means we can't accept contributions from GitHub accounts not in our organization. @rickecon noted that this could be problematic down the line, if for example we want to encourage state legislative analyst offices to contribute.

@nikhilwoodruff could you summarize the blockers here?

**Comment by MaxGhenis** [comment]
Actually does this limitation apply to openfisca-us? https://github.com/PolicyEngine/openfisca-us/pull/675 seems to have gone through fine from a fork.

**Comment by nikhilwoodruff** [comment]
Should the pip-deploying actions be happening on anything other than upstream/master?

**Comment by nikhilwoodruff** [comment]
This is now fixed - contributors can file PRs from forks.
